gui/elements/qt/_viewer.py

```diff
@@ -52,7 +56,7 @@ def update_slider(self, axis, max_axis_length):
         grid = self.layout()
         row = self.viewer._axis_to_row(axis)
 
-        slider = grid.itemAt(row)
+        slider = grid.itemAt(row+1)
```

Comment:
This should be changed in `viewer._axis_to_row` itself.

Reply:
Done